tools/testing/target_determination/heuristics/mentioned_in_pr.py

```diff
@@ -0,0 +1,102 @@
+import json
+import os
+import re
+import subprocess
+from typing import Any, List
+from urllib.request import Request, urlopen
+
+from tools.testing.target_determination.heuristics.interface import (
+    HeuristicInterface,
+    TestPrioritizations,
+)
+from tools.testing.test_run import TestRun
+
+GITHUB_TOKEN = os.environ.get("GITHUB_TOKEN", "")
+
+
+# This heuristic gives a test a rating of 1 if it is mentioned in the PR or a
```

Comment:
also searching in issue body for issues linked in PR title?

Reply:
edited comment to be more clear and have examples